webapp/graphite/logger.py

```diff
@@ -55,6 +55,12 @@ def __init__(self):
                                                level=logging.INFO,
                                                )
 
+    self.warnLogger = self._config_logger('warn.log',
```

Comment:
based on the naming of the loggers defined above, it looks like they opt to be specific rather than generic. thoughts on renaming this to `queryLogger` or `slowQueryLogger`?